### PR TITLE
NIFI-1513: fixed some easy to fix errors

### PR DESCRIPTION
**Pull request by trkurc**

### CODE REVIEW

File: nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GetHTTP.java [hunk 1]
```diff
@@ -426,60 +423,65 @@ public void onTrigger(final ProcessContext context, final ProcessSessionFactory
             if (accept != null) {
                 get.addHeader(HEADER_ACCEPT, accept);
             }
+            // create the http client
+            try ( final CloseableHttpClient client = clientBuilder.build() ) {
+                // NOTE: including this inner try in order to swallow exceptions on close
+                try {
```
Comment (trkurc) [inline nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GetHTTP.java]: Holy smokes, this got butchered, it should just be an indentation of that try block

File: nifi-commons/nifi-processor-utilities/src/main/java/org/apache/nifi/processor/util/bin/Bin.java [hunk 2]
```diff
@@ -121,8 +125,9 @@ public boolean offer(final FlowFile flowFile, final ProcessSession session) {
             final String countValue = flowFile.getAttribute(fileCountAttribute);
             final Integer count = toInteger(countValue);
             if (count != null) {
-                this.maximumEntries = Math.min(count, this.maximumEntries);
-                this.minimumEntries = this.maximumEntries;
+                int currentMaxEntries = this.maximumEntries;
+                this.maximumEntries = Math.min(count, currentMaxEntries);
+                this.minimumEntries = currentMaxEntries;
```
Comment (olegz) [inline nifi-commons/nifi-processor-utilities/src/main/java/org/apache/nifi/processor/util/bin/Bin.java]: Not sure why it was necessary (other then style), but overall PR looks good.
+1

**Comment by mcgilman** [comment]
Reviewing

**Comment by mcgilman** [comment]
I additionally addressed some whitespaces at the end of lines causing contrib-check to fail.

**Pull request closed without merging**